### ISSUE TITLE
Question about how to use the sensu::check

### ISSUE THREAD

**Issue by chaoranxie** [sensu/sensu-puppet#59]
Hi, i am trying to setup a sensu::check with customized attribute, but the format of the json file generated does not seem to work, not sure if this is even a support feature or i am doing something wrong

<pre>
sensu::check { 'check_disk':
    command     => 'PATH=$PATH:/usr/lib64/nagios/plugins:/usr/lib/nagios/plugins check_disk  -w :::check_disk.disk.warning:::% -c :::check_disk.disk.critial:::%',
    handlers    => ["default", "email"],
    interval    => 10,
    config      => {'disk' => { warning => 70,
                                critial => 90,
                              }
                   }  }
</pre>

basically i want to for each node, passing in a different values for warning and critical. but the json file generated from above code looks like

<pre>
cat check_disk.json
{
  "check_disk": {
    "disk": {
      "warning": "70",
      "critial": "90"
    }
  }
}
</pre>

but it needs to be like following for the attribute to read correctly

<pre>
cat check_disk.json
{
  "client": {
    "disk": {
      "warning": "70",
      "critial": "90"
    }
  }
}
</pre>

Am i doing something wrong? Thanks
